SocialCareCaseViewerApi.Tests/V1/Data/MongoDB/MongoDBImportTests.cs

```diff
@@ -0,0 +1,112 @@
+//        {
+//            string mongoConnectionString = Environment.GetEnvironmentVariable("SCCV_MONGO_CONN_STRING") ??
+//                                           Environment.GetEnvironmentVariable("MONGO_CONN_STRING") ??
+//                                           @"mongodb://localhost:1433/";
+
+//            string databaseName = Environment.GetEnvironmentVariable("SCCV_MONGO_DB_NAME") ?? "social_care_db_name";
+
+//            var mongoClient = new MongoClient(new MongoUrl(mongoConnectionString));
+//            _mongoDatabase = mongoClient.GetDatabase(databaseName);
+//        }
+
+//        [TearDown]
+//        public void TearDown()
+//        {
+//            _mongoDatabase?.DropCollection(MainFormCollection);
+//        }
+
+//        [Test]
+//        public void DeletesRecordsMarkedForDeletion()
+//        {
+//            var casenoteOne = _fixture.Build<MongoDBTestObject>().With(x => x.MarkedForDeletion, "true").Create();
+
+//            var collection = _mongoDatabase?.GetCollection<MongoDBTestObject>(MainFormCollection);
+
+//            collection?.InsertOne(casenoteOne);
+
+//            //verify filter
+//            var filterByMarkedForDeletionFlag = Builders<MongoDBTestObject>.Filter.Eq("MarkedForDeletion", "true");
+//            var originalMatchingRecords = collection.Find(filterByMarkedForDeletionFlag).ToList();
+//            originalMatchingRecords.Count.Should().Be(1);
+
+//            var result = _mongoDatabase?.RunCommand(_dbCommandUnderTest);
+
+//            var newMatchingRecords = collection.Find(filterByMarkedForDeletionFlag).ToList();
+//            newMatchingRecords.Count.Should().Be(0);
+//        }
+
+//        [Test]
+//        public void DoesNotDeleteRecordsNotMarkedForDeletion()
+//        {
+//            var casenoteOne = _fixture.Build<MongoDBTestObject>().With(x => x.MarkedForDeletion, "true").Create();
+//            var casenoteTwo = _fixture.Build<MongoDBTestObject>().With(x => x.MarkedForDeletion, "false").Create();
+//            var casenoteThree = _fixture.Create<MongoDBTestObject>();
+
+//            var collection = _mongoDatabase?.GetCollection<MongoDBTestObject>(MainFormCollection);
+
+//            collection?.InsertOne(casenoteOne);
+//            collection?.InsertOne(casenoteTwo);
+//            collection?.InsertOne(casenoteThree);
+
+//            var result = _mongoDatabase?.RunCommand(_dbCommandUnderTest);
+
+//            var filterByMarkedForDeletionFlag = Builders<MongoDBTestObject>.Filter.Eq("MarkedForDeletion", "true");
+//            var newMatchingRecords = collection.Find(filterByMarkedForDeletionFlag).ToList();
+//            newMatchingRecords.Count.Should().Be(0);
+
+//            var unchangedRecordsFilter = Builders<MongoDBTestObject>.Filter.Ne("MarkedForDeletion", "true");
+//            var unchangedRecords = collection.Find(unchangedRecordsFilter).ToList();
+//            unchangedRecords.Count.Should().Be(2);
+//        }
+//    }
+//}
```

Comment:
Should we remove a commented out file for now? Large sections of commented code just add confusion for other developers.

Reply:
I agree, I was hoping to be able to enable that before merging to master, but there have been some delays on the dependencies outside this solution. To keep the master clean it's best to remove that fow now.